src/spglib.h

```diff
@@ -198,7 +220,7 @@ SpglibDataset *spg_get_dataset(const double lattice[3][3],
                                const double position[][3], const int types[],
                                const int num_atom, const double symprec);
 
-/* This is for test using */
+SPG_DEPRECATED("This was not supposed to have been exported")
 SpglibDataset *spg_get_layer_dataset(const double lattice[3][3],
```

Comment:
`spg_get_layer_dataset` is a kind of experimental not deprecated.

Reply:
Yes, I consider it is experimental. In the next release it will be removed. We need to consider better algorithm and practical API.

Reply:
There is no `experimental` attribute, so I will just use deprecated for that here as well.